assets/scss/footer.scss

```diff
@@ -1,75 +1,23 @@
-.nhsuk-footer {
-
-	&__list {
-		width: 100%;
-		margin-bottom: 16px;
-		margin-bottom: 1rem;
-	}
-
-	&__copyright {
-		float: unset;
-		text-align: left;
-    color: #454A4C;
-    width: 100%;
-
-    a {
-      color: #454A4C;
-
-      &:visited {
-        color: #454A4C;
+.hale-page {
+  .govuk-footer .widget_social_widget {
+    //the styles here are just for the social media icon links.
+    font-size: 40px;
+    
+    .hale-social-link {
+      margin-right:12px;
+      
+      &:hover {
+        box-shadow: 0 4px $govuk-text-colour;
       }
+      &:focus {
+        box-shadow: -4px 0px $govuk-focus-colour, 4px 0px $govuk-focus-colour, -4px 4px $govuk-text-colour, 4px 4px $govuk-text-colour;
+      }
+    }
+    .widgettitle {
+      @extend .govuk-heading-s;
     }
-
-		&_img {
-
-			text-align: center;
-
-			img {
-				max-width: 50%;
-			}
-		}
-	}
-	.widget_social_widget {
-		font-size: 2.5rem;
-
-		a {
-			color: #454A4C;
-			margin-right: 12px;
-
-			&:focus, &:hover, &:active  {
-				background-color: transparent;
-				box-shadow: none;
-				color:  $color_marine-blue;
-			}
-		}
-
-		.widgettitle {
-			font-size: 24px;
-			font-size: 1.5rem;
-		}
-	}
+  }
+  .hale-footer__copyright {
+    margin-top: 1em;
```

Comment:
Do we need to have a px fallback? What's the reasoning for using em rather than rem? We've tended to use rem throughout to avoid any potential compounding (maybe that's overcautious)